plugin.py

```diff
@@ -66,6 +66,21 @@ def binary_path(cls) -> str:
         else:
             return os.path.join(cls.basedir(), "omnisharp", "OmniSharp.exe")
 
+    @classmethod
+    def get_solution_arguments(cls) -> List[str]:
+        view = sublime.active_window().active_view()
+        project_file = view.window().project_file_name()
+
+        if project_file is not None:
+            data = view.window().project_data()
+            if 'solution_file' in data:
+                project_dir = os.path.dirname(project_file)
```

Comment:
You shouldn't be assuming that the solution file location is relative to the project file location. Project file can be stored anywhere really. I would rather resolve relative location to the location of the (first) project folder.

And if the location is absolute then none of this path joining should be performed because it wont' be correct in that case.